mbedtls/src/x509/certificate.rs

```diff
@@ -253,6 +254,44 @@ impl Certificate {
         }
         result.map(|_| ())
     }
+
+    pub fn verify_callback<F>(
```

Comment:
Not sure if we're following some pre-existing function naming scheme, but otherwise `verify_with_callback()` seems a more intuitive name to me.

Reply:
I agree that it's a better function name and changed it for now. I'm also not that familiar with the code base. Somebody with more experience may want to pitch in.